docker-compose.yml

```diff
@@ -14,5 +14,20 @@ services:
     volumes:
       - entity-service-postgres:/var/lib/postgresql/data
 
+  mysql:
+    image: mysql
+    container_name: universe-simulator-entity-service-mysql
+    restart: always
+    environment:
+      - MYSQL_DATABASE=${US_ES_MYSQL_DATABASE}
+      - MYSQL_ROOT_PASSWORD=${US_ES_MYSQL_ROOT_PASSWORD}
+      - MYSQL_USER=${US_ES_MYSQL_USER}
+      - MYSQL_PASSWORD=${US_ES_MYSQL_PASSWORD}
+    ports:
+      - ${US_ES_MYSQL_PORT}:3306
+    volumes:
+      - entity-service-mysql:/var/lib/mysql/data
```

Comment:
Mysql container volume name is incorrect. Should be /var/lib/mysql. Check https://hub.docker.com/_/mysql -> Where to Store Data section.